js/build.js

```diff
@@ -29,7 +29,7 @@
     const testDataObject = {};
     let compiledRowTemplate;
 
-    let rowTemplate = $('<div></div>').append($($rowTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
+    let rowTemplate = $('<div></div>').html($rowTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
```

Comment:
@tonytlwu, `rowTemplate` is never reassigned 

```suggestion
    const rowTemplate = $('<div></div>').html($rowTemplate.html() || '').find('fl-prop[data-path]').each(function(i, el) {
```